### ISSUE TITLE
Tabs Component doesn't have color_scheme prop

### ISSUE THREAD

**Issue by raven-black-dream** [reflex-dev/reflex#1790]
**Describe the bug**
Tabs Component doesn't have color_scheme prop, so the colour of the selected tab will always be blue

**To Reproduce**
Create a Tabs component - it is most notable when the variant is set to solid-rounded
Set color_scheme = "green" 
Observe that the colour doesn't change

 - Code/Link to Repo:

**Expected behavior**

The colour of the tab should change when the color_scheme is set.

**Screenshots**

**Specifics (please complete the following information):**
 - Python Version:
 - Reflex Version:
 - OS: 
 - Browser (Optional):
 
**Additional context**

**Comment by payalpm** [comment]
Identify the Tabs Component and Modify the Component: Inside the Tabs component, you will need to modify the code to accept the color_scheme prop and use it to style the selected tab accordingly.

**Comment by raven-black-dream** [comment]
@payalpm 

I did write a pull request to fix it.

**Comment by payalpm** [comment]
Can u brief about this project then i will fix little bit confusing

**Comment by raven-black-dream** [comment]
@payalpm 

Sure. 

I was using the Tabs component, but noticed that I could not change the color of the tabs.

I asked about it on Discord. Lendemor suggested that I write up an issue. I offered to both write up the issue and fix it (if it wasn't too complicated.)

I wrote this issue, and found what I needed to do to fix it. I implemented it and submitted a PR (#1792). That PR is awaiting review.


Thank you,
Evan
